compiler/codegen/src/error.rs

```diff
@@ -1,7 +1,22 @@
 use ruff_source_file::SourceLocation;
-use std::fmt;
+use std::fmt::{self, Display};
 use thiserror::Error;
 
+#[derive(Debug)]
```

Comment:
you may want to try `derive_more::Display` for this simple Display implementation.
This comment is not a change request